aws.yaml

```diff
@@ -551,9 +551,61 @@ kind: Release
 metadata:
   annotations:
     giantswarm.io/docs: https://docs.giantswarm.io/reference/cp-k8s-api/releases.release.giantswarm.io/
-  name: v9.0.2
+  name: v9.0.3
 spec:
   state: active
+  date: 2020-04-17T12:00:00Z
+  apps:
+  - name: cert-exporter
+    version: 1.2.1
+  - name: chart-operator
+    version: 0.12.1
```

Comment:
Also, would like latest chart-operator 0.12.4 in here. https://github.com/giantswarm/chart-operator/blob/master/CHANGELOG.md#v0124-2020-04-15 

cc @rossf7 please confirm if correct :)

Reply:
Yes please update include it. It has several fixes missing in 0.12.1.